static/src/javascripts/projects/common/modules/adfree-survey-simple.js

```diff
@@ -0,0 +1,49 @@
+define([
+    'bean',
+    'fastdom',
+    'common/utils/$',
+    'common/utils/_',
+    'common/utils/template',
+    'common/views/svgs',
+    'text!common/views/commercial/adfree-survey-simple.html'
+], function (
+    bean,
+    fastdom,
+    $,
+    _,
+    template,
+    svgs,
+    adfreeSurveySimple
+) {
+
+    /**
+     * Message which is shown as an overlay to all users who want to remove ads.
+     */
+    var AdfreeSurveySimple = function () {
+    };
+
+    AdfreeSurveySimple.prototype.attach = function () {
```

Comment:
It's not a big deal, but I'm not sure if this actually needs to be on a prototype. An instance method would be slightly more concise than a static.